tests/acceptance/browzine-summon-adapter-test.js

```diff
@@ -50,29 +52,31 @@ describe("BrowZine Summon Adapter", function() {
     });
 
     it("should have an enhanced browzine journal cover", function() {
-      httpBackend.flush();
```

Comment:
Why aren't we flushing between tests anymore?

Reply:
We don't have access to `httpBackend` for now. You get access to the `httpBackend` service when you make AJAX calls in an Angular context (e.g. directive, service, etc.). But we're not able to run in an Angular context until the Summon team loads all scripts before manually bootstrapping angular. Right now in prod, Summon mostly boots before our script loads and when that happens, Angular doesn't know to run our code.

So instead of using `httpBackend` to mock our AJAX calls, I was able to override `$.getJSON` to mock what our API would return. See, https://github.com/thirdiron/browzine-summon-adapter/pull/13/files#diff-a1badd2f04e0cf74e7b88220c9909758R23